auth.go

```diff
@@ -28,11 +23,19 @@ func NewAuthService(client *Client) *AuthService {
 	}
 }
 
-// Login attempts to log a user in with either an Auth0 ID token or a username and password, returning either a (CCloud) token or an error.
-func (a *AuthService) Login(_ context.Context, req *AuthenticateRequest) (*AuthenticateReply, error) {
+// Login attempts to log a user in with an Auth0 ID token, returning either a (CCloud) token or an error.
+func (a *AuthService) Login(ctx context.Context, req *AuthenticateRequest) (*AuthenticateReply, error) {
```

Comment:
I think instead of passing `ctx` to `login` where it's ignored, you could just remove `_ context.Context` entirely from `login` and change `Login`, `OktaLogin` to both use the blank identifier for context.